Source/FieldSolver/SpectralSolver/Make.package

```diff
@@ -5,6 +5,19 @@ CEXE_sources += SpectralFieldData.cpp
 CEXE_headers += SpectralKSpace.H
 CEXE_sources += SpectralKSpace.cpp
```

Comment:
All of the Cartesian spectral code needs to still be included with USE_RZ=TRUE since it is used by the PML code. Maybe a later PR can add ifdefs to the PML code to turn it off with using RZ.